### ISSUE TITLE
From Jim 7/1/2018 - Section 4.1 

### ISSUE THREAD

**Issue by cisco-sro** [SanKumar2015/EST-coaps#75]

**Comment by cisco-sro** [comment]
[JLS]  I do not believe that the wrapping of content with the CBOR binary text wrapping is needed at this point.  If that is needed for the multi-part wrapping, then it is the job of the multi-part wrapping to deal with this problem.  Multipart needs to be able to say I have a multipart of <plain text, json> neither of which are CBOR objects.  Therefor there is no reason for you to use a CBOR wrapper for this and not just use the binary value.

**Comment by cisco-sro** [comment]
How I read this statement is
I have a binary value and I need to look at the content type to figure out what ASN.1 structure it is and decode it.  This is very hard.  So I am going to change this to
I need to look at the content type to figure out that I have a CBOR bstring value.  It is easy to remove the CBOR bstring value.  I now have a binary value and I need to look at the content type to figure out what ASN.1 structure it is and decode it.  This is somehow easier.

**Comment by cisco-sro** [comment]
Right. We don’t need the CBOR binary wrapping. Originally the multipart was not using CBOR either https://tools.ietf.org/html/draft-fossati-core-multipart-ct-03 It was just using the encoding the length and then included the encoded payload. The draft was updated https://tools.ietf.org/html/draft-ietf-core-multipart-ct and to be consistent with CBOR follow this new wrapping methodology that leverages CBOR. What you are saying is right, but we needed to point to a live draft. 

The key and the cert payloads in the multipart conveyed here will be ASN.1 as you suggested, binary encoded. The CT ID for the key and cert are defined in EST-coaps as 280 and 281. We need to make sure we spell them out in the text though to make them clearer.

**Comment by cisco-sro** [comment]
Agreed with Jim, fixed draft subsection and closing. 

"We don’t need the CBOR binary wrapping. Originally the multipart was not using CBOR either https://tools.ietf.org/html/draft-fossati-core-multipart-ct-03 It was just using the encoding the length and then included the encoded payload. The draft was updated https://tools.ietf.org/html/draft-ietf-core-multipart-ct and to be consistent with CBOR follow this new wrapping methodology that leverages CBOR. What you are saying is right, but we needed to point to a live draft. 

The key and the cert payloads in the multipart conveyed here will be ASN.1 as you suggested, binary encoded. The CT ID for the key and cert are defined in EST-coaps as 280 and 281. We need to make sure we spell them out in the text though to make them clearer. "
